pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -1619,28 +1631,20 @@ fn failed_execution() {
             amount
         );
 
+        // Ensure balances have not changed.
         assert_eq!(Asset::balance_of(&ticker, alice_did), alice_init_balance);
         assert_eq!(Asset::balance_of(&ticker, bob_did), bob_init_balance);
         assert_eq!(Asset::balance_of(&ticker2, alice_did), alice_init_balance2);
         assert_eq!(Asset::balance_of(&ticker2, bob_did), bob_init_balance2);
 
+        ensure_instruction_status(instruction_counter, InstructionStatus::Pending);
+
+        // Instruction should execute on the next block and settlement should fail.
```

Comment:
```suggestion
        // Instruction should execute on the next block and settlement should fail,
        // since the tokens are still locked for settlement execution.
```